### ISSUE TITLE
FPPv8:  Backup to remote FPP, WARNINGG Rsync server is disabled message appears event though rsync is enable.

### ISSUE THREAD

**Issue by patdelaney** [FalconChristmas/fpp#1901]
**FPP Version and Hardware**: 
*FPP 8.x master on Pi*

**Describe the bug and Steps to reproduce**: 
When I try to run a file copy backup to a remote FPP instance. I get a warning in the UI. 
WARNING! Rsync server is disabled on remote host, please enable rsync under FPP Settings -> System -> OS Services on 
But the Rsync service is running.. I don't think that the UI is actually running the test. If I click copy, it will copy the files to the remote FPP.

**Expected behavior**: 
There should be no warning if the rsync is actually running.

**Additional Attachments**
![image](https://github.com/user-attachments/assets/5012a70c-284d-4e64-b22b-324f3b451412)

**Comment by OnlineDynamic** [comment]
@patdelaney the warning ip and selected remote host ip address look different in your screen shot?

**Comment by patdelaney** [comment]
This is the sa,e Pi, in "Show network mode"  Meaning Wifi is connected to my Home network and eth0 is connected to the show network.
192.168.1.250 (wifi home)
192.168.50.200 (eth0 show)

**Comment by patdelaney** [comment]
This error only apparently happes on this FPP (Fpp-disk)  I'm not sure whe it wants to check services on 192.168.50.200.  That IP is the gateway for the 192.168.50.x network

**Comment by darylc** [comment]
@patdelaney is the rsyncd process actually running after you enable rsync?
ps auxww|grep rsync

**Comment by darylc** [comment]
@patdelaney ?

**Comment by OnlineDynamic** [comment]
on the remote what does api/settings/Service_rsync report?  the logic for the warning comes from that api call - did this api even exist back in v3/4 ?

**Comment by darylc** [comment]
@patdelaney

**Comment by patdelaney** [comment]
@darylc  yes rsync is running on the receiving Pi. So this issue appear to be that the logic and the data printed to the screen don't match.
In this scenario: the recieving FPP has rsync already running, On the FPP-Disk which is the unit I am trying to back up Rsync was not running and generating the error. But the error says remote server. but is not really the host.  Once I enable rsync on FPP-disk (the sending unit)  The error goes away.


![image](https://github.com/user-attachments/assets/8b1f3fc6-38a2-4ee9-88ca-a162220fa83c)

**Comment by darylc** [comment]
@patdelaney Can you answer Stuart's question above?

**Comment by patdelaney** [comment]
I just retested and this appears to be fixed. I no longer get the warning.

**Comment by patdelaney** [comment]
I ran the command that @OnlineDynamic  asked for:

{
  "name": "Service_rsync",
  "description": "Enable rsync",
  "gatherStats": true,
  "tip": "The rsync daemon is used to receive synced files from other FPP systems.  You must enable rsync on the \u003Cb\u003Edestination\u003C/b\u003E FPP system if you are using the MultiSync page to copy sequences, media, OS, and other files between FPP instances.",
  "type": "checkbox",
  "default": "0",
  "platforms": [
    "!MacOS"
  ],
  "value": "1"
}
